apps/bare-expo/ios/Podfile

```diff
@@ -20,7 +23,7 @@ abstract_target 'BareExpoMain' do
   use_react_native!(
     :path => config[:reactNativePath],
     # to enable hermes on iOS, change `false` to `true` and then install pods
-    :hermes_enabled => false
+    :hermes_enabled => expo_config['jsEngine'] == 'hermes'
```

Comment:
Maybe bind this to a namespace so the args can be passed directly?
```
{
  "react-native": {
    "hermes_enabled": true
  }
}
```

Reply:
just want to align the naming we did in android gradle.properties and app.json. does it make sense?